server/src/main/java/org/spine3/server/aggregate/Aggregate.java

```diff
@@ -349,15 +399,18 @@ void restore(Snapshot snapshot) {
         // See if we're in the state update cycle.
         final B builder = this.builder;
 
-        // If the call to restore() is made during a reply
-        // (because the snapshot event was encountered)
-        // use the currently initialized builder.
+        /* If the call to restore() is made during a replay
+           (because the snapshot event was encountered)
```

Comment:
Not quite. It's stored along with other events. And some people call it a special kind of event. But I agree it may be confusing.